src/state/tutorial/reducer.ts

```diff
@@ -0,0 +1,19 @@
+import { createReducer } from '@reduxjs/toolkit'
+
+import { setShowTutorial } from './actions'
+
+interface TutorialState {
+  swap: { show: boolean; step: number } // in the future we can have many tutorial per page
```

Comment:
`overview` is better name IMO

Reply:
I store it by page. That is the reason why I named it `swap`.  In the next few days. There will have some tutorials like that, per page.

Reply:
make sense